test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -1,490 +1,490 @@
-        pbs_exec = self.server.pbs_conf['PBS_EXEC']
-        pbsnodes_cmd = os.path.join(pbs_exec, 'bin', 'pbsnodes')
-        pbsnodes_offline = [pbsnodes_cmd, '-o', momA.shortname]
-        self.du.run_cmd(self.server.hostname, cmd=pbsnodes_offline)
-
-        # MomA should be offline
-        self.server.expect(
-            NODE, {ATTR_NODE_state: 'offline'},
-            id=momA.shortname, interval=2)
-
-        # momB and the rest of the vnodes should be free
-        self.server.expect(NODE, {ATTR_NODE_state: 'free'},
-                           id=momB.shortname, interval=2)
-
-        self.verify_vnodes_state('free')
-
-    def test_pbsnodes_multi_mom_offline_online(self):
-        """
-        When all of the moms reporting a vnode are offline,
-        the vnode should also be marked offline.
-        And when pbsnodes -r is used to clear the offline from at
-        least one of the moms reporting a vnode, then that vnode
-        should also get the offline cleared.
-        Note: This test needs moms to report the same set of vnodes.
-        """
-        if len(self.moms) != 2:
-            self.skipTest("Provide 2 moms while invoking test")
-
-        for m in self.moms.values():
-            if m.is_cpuset_mom():
-                self.skipTest("Skipping test on cpuset moms")
-
-        momA = self.moms.values()[0]
-        momB = self.moms.values()[1]
-
-        # The moms provided to the test may have unwanted vnodedef files.
-        if momA.has_vnode_defs():
-            momA.delete_vnode_defs()
-        if momB.has_vnode_defs():
-            momB.delete_vnode_defs()
-
-        self.create_multi_vnodes(2)
-
-        # Offline both of the moms, the vnodes reported by them
-        # will also be offlined
-        pbs_exec = self.server.pbs_conf['PBS_EXEC']
-        pbsnodes_cmd = os.path.join(pbs_exec, 'bin', 'pbsnodes')
-        pbsnodes_offline = [pbsnodes_cmd, '-o', momA.shortname, momB.shortname]
-        self.du.run_cmd(self.server.hostname, cmd=pbsnodes_offline)
-
-        # MomA and MomB should be offline
-        self.server.expect(
-            NODE, {ATTR_NODE_state: 'offline'},
-            id=momA.shortname, interval=2)
-
-        self.server.expect(
-            NODE, {ATTR_NODE_state: 'offline'},
-            id=momB.shortname, interval=2)
-
-        self.verify_vnodes_state('offline')
-
-        # Now call pbsnodes -r to clear the offline from MomA
-        pbsnodes_clear_offline = [pbsnodes_cmd, '-r', momA.shortname]
-        self.du.run_cmd(self.server.hostname, cmd=pbsnodes_clear_offline)
-
-        # MomB should still be offline
-        self.server.expect(
-            NODE, {ATTR_NODE_state: 'offline'},
-            id=momB.shortname, interval=2)
-
-        # momA and the vnodes she reports should be free
-        self.server.expect(NODE, {ATTR_NODE_state: 'free'},
-                           id=momA.shortname, interval=2)
-
-        self.verify_vnodes_state('free')
+# coding: utf-8
```

Comment:
why is this whole file showing up as changed? what line did you change in this file?

Reply:
previously file had CRLF line ending when I changed file my editor converted all line endings to LF. I changed line no. 133, removed args from cleanup_jobs()